### PR TITLE
Fixes weird sizing of message bubbles. Closes #1682

### LINKED ISSUE #1682: Swift bug: Messages' widths are not consistent
I have seen this issue with my program (in Swift) for a while. Thus I was very happy to see the new code SwiftExample. Unfortunately that code has the same issue with mine when widths of messages may be changed.

To reproduce: 
- image 1 is the screen of the device in portrait from the beginning
- image 2 is the screen after rotating the device into landscape then rotating back to portrait 

simulator screen shot 20 jun 2016 10 56 55 pm
simulator screen shot 20 jun 2016 10 56 47 pm

### PR DESCRIPTION
**Pull request by MacMeDan**
## Pull request checklist

- [X] All tests pass. Demo project builds and runs.
- [X] I have resolved any merge conflicts.
- [X] I have followed the [coding style](https://github.com/jessesquires/HowToContribute#style-guidelines), and reviewed the [contributing guidelines](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/CONTRIBUTING.md). Confirmation: 💪😎👊

## What's in this pull request?

>Changed Setting label to better reflect what It does. Fixed Message bubble color for tailless images. Fixed sizing of bubbles inconsistency.

### CODE REVIEW

File: SwiftExample/SwiftExample/ChatViewController.swift [hunk 1]
```diff
@@ -51,8 +51,11 @@ class ChatViewController: JSQMessagesViewController {
         
         //Get Messages
         self.messages = makeConversation()
-        self.collectionView?.reloadData()
-        self.collectionView?.layoutIfNeeded()
+        self.collectionView?.setNeedsLayout()
+        self.collectionView?.layoutSubviews()
```
Comment (jessesquires) [inline SwiftExample/SwiftExample/ChatViewController.swift]: `layoutSubviews` should *never* be called directly

**Comment by MacMeDan** [comment]
mmm Ok I will look further into why this issue is happening.

**Comment by jessesquires** [comment]
👍 cool, going to close this out for now. let's open a new PR if we find another fix

**Comment by MacMeDan** [comment]
I swear even apple struggles with this one. I get my last message behind the input bar in messages. 🤗

**Pull request closed without merging**